nekoyume/Assets/_Scripts/UI/Module/Item/ShopItemView.cs

```diff
@@ -1,16 +1,23 @@
-using System.Linq;
-using Nekoyume.TableData;
+using System;
+using System.Collections.Generic;
 using Nekoyume.UI.Model;
 using TMPro;
+using UniRx;
```

Comment:
You can move this line(`using UniRx;`) into namespace below and remove the `using ObservableExtensions = UniRx.ObservableExtensions;` line and fix subscription logic like below.

Before
```
ObservableExtensions
  .Subscribe(Game.Game.instance.Agent.BlockIndexSubject, SetExpired)
  .AddTo(_disposables);
```
After
```
Game.Game.instance.Agent.BlockIndexSubject
  .Subscribe(SetExpired)
  .AddTo(_disposables);
```